src/__tests__/incidents/util/extract-username-util.test.ts

```diff
@@ -0,0 +1,6 @@
+import { extractUsername } from '../../../incidents/util/extractUsername'
+
+it('should extract the string after the last : in a given string', () => {
```

Comment:
a convention the project has been following is that all tests are wrapped in a top-level describe. 

For this case it might be something like

```
describe('extract username util', () => {
	it('...', () => {})
})
```